src/Altinn.App.Core/Models/AppTableOptions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Altinn.App.Core.Models
+{
+    /// <summary>
+    /// Represents values to be used in a Table.
+    /// </summary>
+    public class AppTableOptions
```

Comment:
We need to think a bit about the model. The metadata model, and especially the links part, belongs in the Api project. Core should not know about http stuff. This is the response model from the controller, but I think we perhaps need a different model in core. But just keep it for now, it's easy to change later.